src/test/java/core/basesyntax/service/impl/ParserImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.TransactionLine;
+import core.basesyntax.service.Parser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class ParserImplTest {
+    private static final Parser PARSE_LINE = new ParserImpl();
```

Comment:
Parser it is not a constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
    private static final Parser parser = new ParserImpl();
```